extensions/amp-script/0.1/amp-script.js

```diff
@@ -248,10 +248,24 @@ export class SanitizerImpl {
 
     /** @private {!Object<string, boolean>} */
     this.allowedTags_ = getAllowedTags();
-    // For now, only allow built-in AMP components.
+
+    // TODO(choumx): Support opt-in for variable substitutions and forms.
+    // For now, only allow built-in AMP components except amp-pixel...
     this.allowedTags_['amp-img'] = true;
     this.allowedTags_['amp-layout'] = true;
-    this.allowedTags_['amp-pixel'] = true;
```

Comment:
Can we set these to `false` instead of removing them? That way they're not accidentally enabled by default.

Reply:
Done.